### PR TITLE
Add minimal resolution period

### LINKED ISSUE #671: Resolution Period
We should be setting a global minimum for the Resolution Period, so that Seller can't create an offer that automatically defaults/timesout the buyer, and sends all of the monies to the Seller.

### PR DESCRIPTION
**Pull request by zajck**
Fix #671

### CODE REVIEW

**Review by anajuliabit — CHANGES_REQUESTED** [review]
Looks good to me

Just a missing natspec

File: contracts/protocol/bases/OfferBase.sol [hunk 1]
```diff
@@ -136,9 +136,10 @@ contract OfferBase is ProtocolBase, IBosonOfferEvents {
             // dispute period must be greater than or equal to the minimum dispute period
             require(_offerDurations.disputePeriod >= limits.minDisputePeriod, INVALID_DISPUTE_PERIOD);
 
-            // dispute duration must be greater than zero
+            // resolution period must be between the minimum and maximum resolution periods
             require(
-                _offerDurations.resolutionPeriod > 0 && _offerDurations.resolutionPeriod <= limits.maxResolutionPeriod,
+                _offerDurations.resolutionPeriod >= limits.minResolutionPeriod &&
+                    _offerDurations.resolutionPeriod <= limits.maxResolutionPeriod,
```
[the remark this answers is not in the document]
Comment (zajck) [reply contracts/protocol/bases/OfferBase.sol:142]: Fixed here and on every other occasion.

**Review by anajuliabit — DISMISSED** [review]

**Review by mischat — APPROVED** [review]
These look good to me. Thanks

**Comment by mischat** [comment]
i have made a note that we will have to update the fair exchange policy to reflect this new minimum period. And I have reached out to the legal / compliance team to see if we need to reflect this change in our Buyer / Seller agreement.

**Pull request merged**